sdk/include/opentelemetry/sdk/trace/sampler.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
```

Comment:
This file is just copied from Oliver's PR, right?

Reply:
Yep